packages/components/src/app/BaseApp.tsx

```diff
@@ -60,6 +64,12 @@ function BaseApp({
     [createMatomoInstance]
   );
 
+  const askemFeedbackInstance = React.useMemo(
+    () => createAskemInstance(askemFeedbackConfiguration),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [createAskemInstance]
```

Comment:
Trivial, but I would say that the `askemFeedbackConfiguration` is more important here than the createAskemInstance. Instead of disabling the exhaustive-deps check, I would just add both variables to the deps.